airbyte-config/config-models/src/main/resources/types/StandardSync.yaml

```diff
@@ -46,16 +46,40 @@ properties:
       - active
       - inactive
       - deprecated
-  # Ideally schedule and manual should be a union, but java
-  # codegen does not handle the union type properly.
-  # When schedule is defined, manual should be false.
   schedule:
     type: object
-    required:
-      - timeUnit
-      - units
     additionalProperties: false
```

Comment:
This might be an issue with exisiting connection. I think it should be ok since the object mapper override this value. Can we set it as true in case it is not override.

Reply:
that is a great call out. @mfsiega-airbyte we should create an existing connection, apply these changes, and make sure we are still able to deserialise the existing json blob without errors.

Reply:
Steps I took to test this:
- Checked out master, rebuilt, ran locally.
- Created a new connection with 15 minute interval, first run succeeded.
- Waited 15 minutes, second run succeeded.
- Stopped the service, checked out msiega/cronstrings, rebuilt, re-ran locally.
- Third run occurred 23 minutes after the second, successful.

This seems to indicate that it can still deserialise fine? The only slightly curious thing is the extra 8 minutes, but I guess that's a result of how scheduling works/restarting the service?

Reply:
agree it looks like we can deserialise fine. 

the 8 mins interval is strange. Do you know if this is a consistent delay?

Reply:
Thanks for testing it, can we still change the additional property to `true`.

Reply:
Not strongly opposed to changing it, but curious as to why? I would expect `additionalProperties: false` to be the preferred default unless there's a particular reason otherwise.

Note that the latest version of this PR doesn't touch the definition/usage of the `schedule` property at all.